src/main/java/spoon/reflect/code/UnaryOperatorKind.java

```diff
@@ -23,34 +23,47 @@
 	/**
 	 * Positivation.
 	 */
-	POS, // +
+	POS(true), // +
 	/**
 	 * Negation.
 	 */
-	NEG, // -
+	NEG(true), // -
 	/**
 	 * Logical inversion.
 	 */
-	NOT, // !
+	NOT(true), // !
 	/**
 	 * Binary complement.
 	 */
-	COMPL, // ~
+	COMPL(true), // ~
 	/**
 	 * Incrementation pre assignment.
 	 */
-	PREINC, // ++ _
+	PREINC(true), // ++ _
 	/**
 	 * Decrementation pre assignment.
 	 */
-	PREDEC, // -- _
+	PREDEC(true), // -- _
 	/**
 	 * Incrementation post assignment.
 	 */
-	POSTINC, // _ ++
+	POSTINC(false), // _ ++
```

Comment:
I'd say we already have the postfix information in the naming convention. If it starts with POST it's postfixed, else if's prefixed. Correct?

Reply:
Will this be OK from your point of view?
```java
UnaryOperatorKind() {
  this.prefixOperator = name().startsWith("POST")==false;
}
```